docs/overview/getting-started/avalanche-consensus.md

```diff
@@ -2,21 +2,49 @@
 description: Consensus is the task of getting a group of computers to come to an agreement on a decision. Avalanche is a new consensus protocol that is scalable, robust, and decentralized.
 ---
 
-# Avalanche Blockchain Consensus
+# Consensus
 
-Consensus is the task of getting a group of computers to come to an agreement on a decision.
-Computers can reach a consensus by following a set of steps called a consensus protocol. Avalanche
+## Avalanche Consensus
+
+Consensus is the task of getting a group of nodes to come to an agreement on a decision.
+Nodes can reach a consensus by following a set of steps called a consensus protocol. Avalanche
 is a new consensus protocol that is scalable, robust, and decentralized. It has low latency and high
 throughput. It is energy efficient and does not require special computer hardware. It performs well
 in adversarial conditions and is resilient to "51% attacks." This document explains the Avalanche
-consensus protocol. The white-paper is [here.](https://www.avalabs.org/white-papers)
+consensus protocol. The white-paper is [here](https://www.avalabs.org/white-papers).
+
+![Consensus Comparison](/img/Consensus-protocol-comparison.png)
+
+Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
+[validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
+determining whether a
+[transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
+accepted or rejected, it asks a small, random subset of validators whether they think the
+transaction should be accepted or rejected. If the queried validator thinks the transaction is
+invalid, has already rejected the transaction, or prefers a conflicting transaction, it replies that
+it thinks the transaction should be rejected. Otherwise, it replies that it thinks the transaction
+should be accepted.
+
+If a sufficiently large portion (_alpha_ α) of the validators sampled reply that they think the
+transaction should be accepted, the validator prefers to accept the transaction. That is, when it is
+queried about the transaction in the future, it will reply that it thinks the transaction should be
+accepted. Similarly, the validator will prefer to reject the transaction if a sufficiently large
```

Comment:
Can we make it more clear that validators vote positively? Consensus will only reject transactions as a result of accepting a conflict. This makes it sound like the network keeps a confidence counter of the number of times in a row the network has voted to reject a transaction, which is not quite accurate.

Reply:
Bumping this comment

Reply:
Yeah this was copy-pasta'd from the existing consensus doc on the intro page, how else should we word this? 

> If a sufficiently large portion (alpha α) of the validators sampled reply that they think the transaction should be accepted, the validator prefers to accept the transaction. Similarly, the validator will prefer to reject the transaction if a sufficiently large portion of the validators replies that they think the transaction should be rejected. After completing a round of sampling, the validator's preference determines how it responds when it, in turn, is sampled by other validators.